### ISSUE TITLE
YANG schema definition not fully consistent in package draft and version selection draft

### ISSUE THREAD

**Issue by lana-wu** [netmod-wg/yang-ver-dt#38]
For the package draft, section 1 Terminology:

o  YANG schema: A datastore schema, not bound to any particular datastore.

Different definition in the version selection draft:
   o  YANG schema: The combined set of schema nodes for a set of YANG
      module revisions, taking into consideration any deviations and
      enabled features.

Rob comments:
I think that both definitions really amount to the same thing, noting the RFC 8342 defines datastore schema as:
   o  datastore schema: The combined set of schema nodes for all modules
      supported by a particular datastore, taking into consideration any
      deviations and enabled features for that datastore.

Basically, every package represents a schema (that might be incomplete).  In some cases that schema may also represent a datastore schema, or potentially a union of compatible packages represents a datastore schema.

But I think that the second definition is potentially better. 

I think that we should get an agreed definition into the packages draft, and then reference that from the version selection draft.

**Comment by schoenw** [comment]
Sound to me like a package defines a 'package schema' and one or multiple package schemas (schemata?) may define a 'datastore schema', i.e, a 1:n relationship. I would be careful with defining something like 'YANG schema'. RFC 8525 defines 'schema' as a synonym for datastore
schema. (Not sure why we would need a second synonym.)

**Comment by rgwilton** [comment]
Yes, when you put it like that, calling it a "package schema" does seem like a rather obvious name. :-)

**Comment by jsterne** [comment]
Rob will propose specific text changes

**Comment by jsterne** [comment]
Overlaps with Issue #74 items (a) and (c).  See email on NETMOD WG list with subject "Definitions of YANG schema and Packages (in packages draft) - Issue #74".
